x-pack/solutions/security/plugins/security_solution/server/lib/detection_engine/rule_types/indicator_match/threat_mapping/utils.ts

```diff
@@ -127,12 +128,14 @@ export const combineConcurrentResults = (
         createdSignals: [...accum.createdSignals, ...item.createdSignals],
         warningMessages: [...accum.warningMessages, ...item.warningMessages],
         errors: [...new Set([...accum.errors, ...item.errors])],
+        userError: accum.userError || item.errors.some((err) => checkErrorDetails(err).isUserError),
```

Comment:
@marshallmain similar logic to create security rule type wrapper. I'm assuming we want this to be `items.errors.every` instead of `.some()` right?